anime-list-master.xml

```diff
@@ -12532,8 +12532,12 @@
   <anime anidbid="4245" tvdbid="hentai" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>True Blue Gaiden</name>
   </anime>
-  <anime anidbid="4247" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="4247" tvdbid="78722" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Cubix</name>
+	<mapping-list>
+      <mapping anidbseason="1" tvdbseason="1" start="1" end="13"/>
+      <mapping anidbseason="1" tvdbseason="2" start="14" end="26" offset="-13"/>
+    </mapping-list>
```

Comment:
Opening is over-indented. The list uses spaces, not tabs
First part is redundant since 1>1 is assumed by default. It doesn't hurt either but adds an extra unnecessary line
```suggestion
    <mapping-list>
      <mapping anidbseason="1" tvdbseason="2" start="14" end="26" offset="-13"/>
    </mapping-list>
```

Reply:
Removed tab and unnecessary line.